packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -38,6 +39,18 @@ The purpose for enabling grouped batching on top of compression is that regular
 
 See [below](#how-grouped-batching-works) for an example.
 
+### Risks
+
+This option should **ONLY** be enabled after observing that 99.9% of your application sessions contains these changes (version "2.0.0-internal.4.1.0" or later). This option is experimental and should not be enabled yet in production. Containers created with this option may not open in future versions of the framework.
```

Comment:
Does this refer to **runtime** 2.0.0-internal.4.1.0? Whatever piece(s) it refers to, it's probably worth it to make it explicit.

I think I'd also start with the "this is experimental, don't enable in production" bit and then follow up with "if you want to test this in other environments, make sure that 99.9+% of you application sessions are using version `x.y.z`".